lib/src/messages.dart

```diff
@@ -43,6 +43,13 @@ class Message {
 
   Message(this.level, this.message, {this.span, this.useColors = false});
 
+  String get describe {
+    if (span == null) return message;
+
+    var start = span!.start;
```

Comment:
Shadow the field with a local to enable type promotion on null check:

```suggestion
    var span = this.span;
    if (span == null) {
      return message;
    }
    
    var start = span.start;
```